modules/nw-kuryr-cleanup.adoc

```diff
@@ -14,7 +14,7 @@ plugin, you must clean up the resources that Kuryr created previously.
 The clean up process relies on a Python virtual environment to ensure that the package versions that you use support tags for Octavia objects. You do not need a virtual environment if you are certain that your environment uses at minimum:
 * `openstacksdk` version 0.54.0
 * `python-openstackclient` version 5.5.0
-* `python-octaviaclient` version 2.3.0 
+* `python-octaviaclient` version 2.3.0
```

Comment:
Can't directly make suggestions on deleted lines for whatever reason, but I think this needs to be restructured to make the list work.

```
The clean up process relies on a Python virtual environment to ensure that the package versions that you use support tags for Octavia objects. You do not need a virtual environment if you are certain that your environment uses at minimum:

* `openstacksdk` version 0.54.0

* `python-openstackclient` version 5.5.0

* `python-octaviaclient` version 2.3.0
```

Reply:
oh, right.